libs/output-mapping/src/Mapping/MappingFromConfigurationDeleteWhereFilterFromWorkspace.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Mapping;
+
+class MappingFromConfigurationDeleteWhereFilterFromWorkspace extends AbstractMappingFromConfigurationDeleteWhereFilter
+{
+    public function getWorkspaceId(): string
+    {
+        return $this->mapping['values_from_workspace']['workspace_id'];
+    }
+
+    public function getWorkspaceTable(): string
+    {
+        return $this->mapping['values_from_workspace']['table'];
+    }
+
+    public function getWorkspaceColumn(): string
```

Comment:
tohle by mělo být nullable, teď se to ale nepoužívá tak je to jedno
opravil jsem to až v samotné implementaci v navazujícím PR https://github.com/keboola/platform-libraries/pull/362/files#r1930060707